addons/custom-fonts/addon.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Custom Fonts",
+    "description": "Change the fonts on Scratch! Uses google's WebFont loader. Only fonts avaliable on google fonts (and from google) can be used. Find the font you like then paste the name into the box!",
```

Comment:
"Uses google's WebFont loader. Only fonts avaliable on google fonts (and from google) can be used. Find the font you like then paste the name into the box!" is a little long...

Suggested new description: "Customize the font. Pick any font from https://fonts.google.com/ and type it below."